packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -90,16 +95,8 @@ export function importGlobPlugin(config: ResolvedConfig): Plugin {
       if (!importGlobMap) return
 
       const modules: EnvironmentModuleNode[] = []
-      for (const [id, allGlobs] of importGlobMap) {
-        // (glob1 || glob2) && !glob3 && !glob4...
-        if (
-          allGlobs.some(
-            ({ affirmed, negated }) =>
-              (!affirmed.length ||
-                affirmed.some((glob) => isMatch(file, glob))) &&
-              (!negated.length || negated.every((glob) => isMatch(file, glob))),
```

Comment:
I moved this code above to make it

- easier to read (the code was split into two parts unnecessary)
- performant (now the glob matcher is precompiled)